### DISCUSSION TITLE
Error in cherry-pick comments

### DISCUSSION THREAD

**Discussion by BhavikaSharma** [argoproj/argo-workflows#9531] [Q&A] [answered]
Hi, 

I noticed in the changelogs for versions `v3.3.7`-[v3.3.9](https://github.com/argoproj/argo-workflows/issues/9262) that the commit https://github.com/argoproj/argo-workflows/commit/aa366db345d794f0d330336d51eb2a88f14ebbe6 was mentioned however I believe that this is only added from `v3.4.0-rc1`. I was wondering if there are any plans to release a `v3.3.10` and/or if it were possible to get this commit into a v3.3.10 release. 

I believe that this would be more involved than just cherry-picking that single commit into the v3.3 release branch though.

Thanks.

**Comment by terrytangyuan** [comment, marked as answer]
That is a bug fix, so I would suggest cherry-picking it for v3.3.10 if we have any plans. cc @sarabala1979 @alexec to comment
